master/buildbot/test/unit/test_worker_docker.py

```diff
@@ -29,6 +29,12 @@
 from buildbot.worker import docker as dockerworker
 
 
+_thd_parse_volumes__orig = dockerworker.DockerLatentWorker._thd_parse_volumes
+def _thd_parse_volumes(self, volumes):
+    self._test_volumes, self._test_binds = retval = _thd_parse_volumes__orig(self, volumes)
```

Comment:
I'm not sure how this would work.

The create_container call is a member of the docker.Client, but the Client object gets instantiated in _thd_start_container and is only available/valid for the duration of the function. Does this mean I have to store the created objects in a global/class variable in order to access/assert it in the test function later on or is there a better way?